interpreter/src/json.rs

```diff
@@ -42,6 +43,7 @@ impl Value {
             Value::Timestamp(ref dt) => dt.to_rfc3339().into(),
             Value::Bytes(ref b) => BASE64_STANDARD.encode(b.as_slice()).to_string().into(),
             Value::Null => serde_json::Value::Null,
+            Value::Duration(v) => format_duration(&v).to_string().into(),
```

Comment:
Hmm... do we serialize the duration to a duration string like `1s`, or do we serialize it to a number of seconds or something? [Go serializes durations to nanoseconds](https://goplay.tools/snippet/cHJDYlgWS8C).